### PR TITLE
2022-02-09-classifierdl_distilbert_sentiment_vi

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      .setPoolingStrategy("AVERAGE")
+
+sentimentClassifier = ClassifierDLModel.pretrained('classifierdl_distilbert_sentiment', 'vi') \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+vi_sentiment_pipeline = Pipeline(stages=[document, tokenizer, normalizer, lemmatizer, distilbert, embeddingsSentence, sentimentClassifier])
+
+light_pipeline = LightPipeline(vi_sentiment_pipeline.fit(spark.createDataFrame([['']]).toDF("text")))
+result = light_pipeline.annotate("Chất cotton siêu đẹp mịn mát.")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md:72]: Is this sentence a real one from the dataset or any other native internet resource?

File: docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+val normalizer = Normalizer()
+      .setInputCols(Array("token"))
+      .setOutputCol("normalized")
+
+val lemmatizer = LemmatizerModel.pretrained("lemma", "vi")
+        .setInputCols(Array("normalized"))
+        .setOutputCol("lemma")
+
+val distilbert = DistilBertEmbeddings.pretrained("distilbert_base_cased", "vi")
+  .setInputCols(Array("document",'token'))
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md:94]: Double quotes required around token

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]

File: docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md:72]: @josejuanmartinez 

it is taken from train data set


image

File: docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2022-02-09-classifierdl_distilbert_sentiment_vi.md:94]: my bad
will fix

**Review by josejuanmartinez — APPROVED** [review]

**Pull request merged**